build/CopyFilesToStagingDir.ps1

```diff
@@ -130,6 +130,19 @@ PublishFile $FullBuildOutput\WindowsAppSDK_DLL\Microsoft.Windows.ApplicationMode
 PublishFile $FullBuildOutput\WindowsAppSDK_DLL\Microsoft.Windows.PushNotifications.winmd $NugetDir\lib\native
 #
 # C#/WinRT Projections
+#TODO:BEGIN @scottj1s @kythant are these right? anycpu\net5.0? net5.0-windows? Should these be net5.0-windows10.0.17763.0 or...?
```

Comment:
@Scottj1s @kythant is this section right? I'm murky on the NuGet subdir targeting for .NETCore 5.0.